server/datastore/mysql/migrations/tables/20230706141219_QueriesTableSchemaChanges.go

```diff
@@ -0,0 +1,49 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230706141219, Down_20230706141219)
+}
+
+func Up_20230706141219(tx *sql.Tx) error {
+	// If we want to drop the uniq constraint on queries.name, we first need to remove this
+	// constraint on scheduled_queries, due to a FK constraint scheduled_queries (query_name) =>
+	// queries (name).
+	if _, err := tx.Exec(`
+		ALTER TABLE scheduled_queries DROP FOREIGN KEY scheduled_queries_query_name;
+	`); err != nil {
+		return errors.Wrap(err, "removing FK on scheduled_queries")
+	}
+
+	if _, err := tx.Exec(`
+		ALTER TABLE queries
+			DROP INDEX idx_query_unique_name,
+			DROP INDEX constraint_query_name_unique,
+
+			ADD team_id INT(10) UNSIGNED DEFAULT NULL,
+			ADD team_id_char CHAR(10) DEFAULT '',
```

Comment:
Could you document the purpose of this column?

Reply:
Added in the type def for better visibility https://github.com/fleetdm/fleet/pull/12673/commits/709f767b89d56d96d55c54051ffbc76ffe44dfa7